.github/workflows/cluster_endtoend_vreplication_fk.yml

```diff
@@ -0,0 +1,161 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
```

Comment:
I will run this test on one of the existing shards once the basic implementation is reviewed.